x11-misc/spacefm/spacefm-1.0.6.ebuild

```diff
@@ -47,42 +49,26 @@ src_configure() {
 		--disable-hal \
 		--enable-inotify \
 		--disable-pixmaps \
-		$(use_with gtk3 gtk3 "yes")
+		--with-gtk3
 }
 
 pkg_postinst() {
-	xdg_desktop_database_update
-	xdg_mimeinfo_database_update
+	xdg_pkg_postinst
 	gnome2_icon_cache_update
```

Comment:
Do note that this wouldn't work with EAPI-7, and should produce an error to your build.log output. Did you test this?

Reply:
I've tested everything. Why would this not work? Ebuild is using gnome2-utils.eclass, not gnome2.eclass, if that's what you're referring to.

Reply:
This command doesn't exist with EAPI-7, 
https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/gnome2-utils.eclass#n449

```
>>> /usr/share/mime/packages/spacefm-mime.xml
 * Updating .desktop files database ...                                                      [ ok ]
 * Updating icons cache ...                                                                  [ ok ]
 * Updating shared mime info database ...                                                    [ ok ]
/var/tmp/portage/gpm/portage/x11-misc/spacefm-1.0.6/temp/environment: line 2080: gnome2_icon_cache_update: command not found
 *   sys-apps/udevil for To mount as non-root user
 *   sys-apps/pmount for To mount as non-root user
 *   sys-fs/udisks for To mount as non-root user
 *   sys-apps/udevil for Support ftp/nfs/smb/ssh URLs in the path bar
 *   x11-misc/ktsuss for To perform as root functionality
 *   kde-plasma/kde-cli-tools[kdesu] for To perform as root functionality
 * When using SpaceFM without udisks, and without the udisks-daemon running,
 * you may need to enable kernel polling for device media changes to be detected.
 * See /usr/share/doc/spacefm-1.0.6/html/spacefm-manual-en.html#devices-kernpoll
>>> x11-misc/spacefm-1.0.6 merged.
```
Besides you already call `xdg_pkg_postinst` so `gnome2_icon_cache_update` should be removed nevertheless.

Reply:
Thanks, I didn't hit this in my log initially. I don't think we need gnome2-utils at all here so I've dropped it.